File: src/com/atl/atlmovil/entidades/DetallePedido.java
package com.atl.atlmovil.entidades;

import java.util.List;

public class DetallePedido {
	private long idPedido;
	private long codigoProducto;
	private double precioUnitario;
	private List<TallaPedido> tallas;
	
	
	public long getIdPedido() {
		return idPedido;
	}
	public void setIdPedido(long idPedido) {
		this.idPedido = idPedido;
	}
	public long getCodigoProducto() {
		return codigoProducto;
	}
	public void setCodigoProducto(long codigoProducto) {
		this.codigoProducto = codigoProducto;
	}
	public double getPrecioUnitario() {
		return precioUnitario;
	}
	public void setPrecioUnitario(double precioUnitario) {
		this.precioUnitario = precioUnitario;
	}
	public List<TallaPedido> getTallas() {
		return tallas;
	}
	public void setTallas(List<TallaPedido> tallas) {
		this.tallas = tallas;
	}
	
	
}

File: net/ATL.Servicio/ATL.Entidad/DetallePedido.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace ATL.Entidad
{
    public class DetallePedido
    {
        public long idPedido { get; set; }
        public long codigoPedido { get; set; }
        public long codigoProducto { get; set; }
        public decimal precioUnitario { get; set; }
        public List<TallaPedido> tallas { get; set; }
	
    }
}
